torchtext/experimental/transforms.py

```diff
@@ -164,25 +163,6 @@ def __prepare_scriptable__(self):
         return RegexTokenizer(regex_tokenizer)
 
 
-class TextSequentialTransforms(nn.Sequential):
```

Comment:
Could you also clean up the tests associated with this Transform. The unit tests are currently failing for this.